test/pubnub/mzcompose.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright Materialize, Inc. All rights reserved.
```

Comment:
You need to register this file in ```ci/test/pipeline.template.yml``` so that it gets run by the CI.

Reply:
done